src/storage/overflow_file.c

```diff
@@ -522,22 +511,15 @@ overflow_update (THREAD_ENTRY * thread_p, const VFID * ovf_vfid, const VPID * ov
       data += copy_length;
       length -= copy_length;
 
-      if (isnewpage)
-	{
-	  log_append_redo_data (thread_p, RVOVF_NEWPAGE_INSERT, &addr, copy_length + hdr_length, addr.pgptr);
-	}
-      else
-	{
-	  log_append_redo_data (thread_p, RVOVF_PAGE_UPDATE, &addr, copy_length + hdr_length, (char *) addr.pgptr);
-	}
+      log_append_redo_data (thread_p, RVOVF_PAGE_UPDATE, &addr, copy_length + hdr_length, (char *) addr.pgptr);
```

Comment:
You are right, it is better to use RVOVF_NEWPAGE_INSERT. The recovery is the same, but log applier can see them differently. Probably RVOVF_UPDATE_PAGE could also be used, but it's more clear this way.

I updated the code.